architecture.md

```diff
@@ -644,9 +644,40 @@ a pod is **not** `Privileged` the runtime selection is done as follows:
 | Default CRI-O trust level: `untrusted` | Kata Containers                               | Kata Containers                              |  Kata Containers |
 
 
-### CRI-containerd
+### Containerd CRI Plugin (Formerly CRI-containerd)
+
+The general guidelines for the Containerd CRI Plugin support is similar to the CRI-O support. You can run trusted workloads with a runtime like runC and then run an untrusted workload with Kata Containers. The parameters that can be modified in the containerd config to run Kata Containers along with another 'trusted' runtime are the following:
```

Comment:
Could you make that `runc` for consistency with the rest of the doc? There are a couple of instances where we aren't rendering that in a fixed font (which you could always fix in a separate commit if you want to ;)